src/main/resources/templates/home.html

```diff
@@ -26,16 +42,16 @@ <h1 class="text-center">Hi, <span th:text="${user.firstName}">User</span>!</h1>
   <!-- Profile Image Section -->
   <div class="text-center mt-4">
     <img th:src="'data:image/jpeg;base64,' + ${profileImage}" alt="Profile Image"
-         class="img-thumbnail rounded-circle profile-image-class" style="width: 150px; height: 150px;" />
+         class="img-thumbnail rounded-circle" style="width: 150px; height: 150px; object-fit: cover;" />
   </div>
 
   <!-- Action Buttons -->
   <div class="text-center mt-4">
-    <a href="/users/profile" class="btn btn-primary me-2 action-button">Edit Profile</a>
-    <a href="/vehicles/" class="btn btn-primary me-2 action-button">Manage Vehicles</a>
-    <a href="/rides" class="btn btn-primary me-2 action-button">View Rides</a>
-    <a href="/rides/create" class="btn btn-primary me-2 action-button">Create Ride</a>
-    <a href="/rides/find" class="btn btn-primary me-2 action-button">Find Rides</a>
```

Comment:
pl add "action-button" class for all 5 links

Reply:
Yes worked on it